apps/protailwind/src/video/exercise-overlay.tsx

```diff
@@ -594,46 +594,66 @@ const handleContinue = async ({
   path,
 }: {
   router: NextRouter
-  module: SanityDocument
-  section?: SanityDocument
-  nextExercise?: LessonResource | null
+  module: Module
+  section?: Section | null
```

Comment:
`Module` and `Section` was the most pervasive change. Had a lot of cascading types